sqldelight-gradle-plugin/src/test/migration-squash-drop-failure/build.gradle

```diff
@@ -0,0 +1,18 @@
+buildscript {
+  apply from: "${projectDir.absolutePath}/../buildscript.gradle"
+}
+
+apply plugin: 'org.jetbrains.kotlin.jvm'
+apply plugin: 'app.cash.sqldelight'
+
+repositories {
+  maven {
+    url "file://${projectDir.absolutePath}/../../../../build/localMaven"
+  }
+}
+
+sqldelight {
+  Database {
+    packageName = "com.example"
+  }
```

Comment:
You also need `deriveSchemaFromMigrations = true` otherwise the squash task is not registered

Reply:
Yes. Thank you. I don't have Android Studio so I actually can't even build locally.

Reply:
You don't need Android Studio, I don't use it too. For debugging just link the project by right click the gradle.build file and test it. Or simple run the test.